test/e2e/framework.go

```diff
@@ -1260,18 +1273,40 @@ func (data *TestData) gracefulExitAntreaAgent(covDir string, nodeName string) er
 		if err != nil {
 			return fmt.Errorf("error when graceful exit Antrea agent: copy pod files out, error:%v", err)
```

Comment:
why does it still say "pod" but you changed it in the `gracefulExitAntreaController` function?

Reply:
also changed this line to the similar in the gracefulExitAntreaController function.